### DISCUSSION TITLE
About batch size

### DISCUSSION THREAD

**Discussion by eceisik** [FETS-AI/Challenge#96] [Q&A] [answered]
Hi,
according to your explanation batches per round is a flat number of batches each training collaborator will train. So I think that batch size is fixed in your codes. Can I learn the batch size? 

Thank you,
Ece

**Comment by sarthakpati** [comment, marked as answer]
Hi,

I think the batch size is hard-coded to `1`.

**Reply by eceisik** [reply]
Thank you.
